hw/scripts/openocd.sh

```diff
@@ -17,6 +17,9 @@
 #
 . $CORE_PATH/hw/scripts/common.sh
 
+CFG_RESET="reset halt"
+GDB=arm-none-eabi-gdb
```

Comment:
Maybe we should add it to compiler.yml file along with other tool related links, and then pass as environment variable to this script. I'll file a Jira ticket about that.